src/EA.Weee.Integration.Tests/Handlers/DirectRegistrant/CompleteOrganisationTransactionHandlerIntegrationTests.cs

```diff
@@ -274,6 +284,105 @@ public class WhenICompleteAPartnershipOrganisationTransaction : CompleteOrganisa
+                organisation.OrganisationName.Should()
+                    .Be(organisationTransactionData.RegisteredCompanyDetailsViewModel.CompanyName);
+                organisation.TradingName.Should()
+                    .Be(organisationTransactionData.RegisteredCompanyDetailsViewModel.BusinessTradingName);
+                organisation.CompanyRegistrationNumber.Should()
+                    .Be(organisationTransactionData.RegisteredCompanyDetailsViewModel.CompaniesRegistrationNumber);
+                organisation.OrganisationStatus.Should().Be(Domain.Organisation.OrganisationStatus.Complete);
+                entity.Organisation.Should().Be(organisation);
+            };
+
+            private readonly It shouldHaveCreatedDirectRegistrant = () =>
+            {
+                var directRegistrant = Query.GetDirectRegistrantByOrganisationId(result);
+
+                directRegistrant.Should().NotBeNull();
+                directRegistrant.BrandName.Name.Should()
+                    .Be(organisationTransactionData.RegisteredCompanyDetailsViewModel.EEEBrandNames);
+                directRegistrant.AuthorisedRepresentative.Should().NotBeNull();
+                directRegistrant.AuthorisedRepresentative.OverseasProducerName.Should().Be(representingCompanyDetails.CompanyName);
+                directRegistrant.AuthorisedRepresentative.OverseasProducerTradingName.Should().Be(representingCompanyDetails.BusinessTradingName);
+                directRegistrant.AuthorisedRepresentative.OverseasContact.Address.PrimaryName.Should().Be(representingCompanyDetails.Address.Address1);
+                directRegistrant.AuthorisedRepresentative.OverseasContact.Address.Street.Should().Be(representingCompanyDetails.Address.Address2);
+                directRegistrant.AuthorisedRepresentative.OverseasContact.Address.Town.Should().Be(representingCompanyDetails.Address.TownOrCity);
+                directRegistrant.AuthorisedRepresentative.OverseasContact.Address.AdministrativeArea.Should().Be(representingCompanyDetails.Address.CountyOrRegion);
+                directRegistrant.AuthorisedRepresentative.OverseasContact.Address.CountryId.Should().Be(representingCompanyDetails.Address.CountryId);
+                directRegistrant.AuthorisedRepresentative.OverseasContact.Address.PostCode.Should().Be(representingCompanyDetails.Address.Postcode);
+                directRegistrant.AuthorisedRepresentative.OverseasContact.Email.Should().Be(representingCompanyDetails.Address.Email);
+                directRegistrant.AuthorisedRepresentative.OverseasContact.Telephone.Should().Be(representingCompanyDetails.Address.Telephone);
+                directRegistrant.AuthorisedRepresentative.OverseasContact.Address.SecondaryName.Should().BeEmpty();
+                directRegistrant.AuthorisedRepresentative.OverseasContact.Address.Locality.Should().BeEmpty();
+                directRegistrant.AuthorisedRepresentative.OverseasContact.SurName.Should().BeEmpty();
```

Comment:
Have we intentionally got these as "SurName" and "ForeName"?

Reply:
@will-hope98  Legacy code, but good spot.